src/service/paranet-service.js

```diff
@@ -44,13 +44,13 @@ class ParanetService {
         return this.blockchainModuleManager.keccak256(blockchain, keyword);
     }
 
-    getParanetRepositoryName(paranetId) {
-        if (this.ualService.isUAL(paranetId)) {
+    getParanetRepositoryName(paranetUAL) {
```

Comment:
We surely use it somewhere else as well, let's not forget to update arguments there

Reply:
This just changed naming of argument in function definition